packages/database/scripts/api-report.ts

```diff
@@ -0,0 +1,178 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as fs from 'fs';
+import * as path from 'path';
+
+import { Extractor, ExtractorConfig } from 'api-extractor-me';
+import * as tmp from 'tmp';
+
+import {
+  pruneDts,
+  removeUnusedImports
+} from '../../../repo-scripts/prune-dts/prune-dts';
+
+// TODOD(mrschmidt): Combine this script with the Firestore version.
+
+/* eslint-disable no-console */
+
+// This script takes the output of the API Extractor, post-processes it using
+// the pruned-dts script and then invokes API report to generate a report
+// that only includes exported symbols. This is all done in temporary folders,
+// all configuration is auto-generated for each run.
+
+const baseApiExtractorConfigFile: string = path.resolve(
```

Comment:
Seems we should share this script across projects, but let's do it in a separate PR.

Reply:
Yup, I thought I addressed this task by adding the TODO :)